docs/Speech and Audio/Getting Microphone Audio/index.md

```diff
@@ -45,6 +109,14 @@ performAPT.setBitsPerSample(BitsPerSample._16_BIT);
 performAPT.setAudioType(AudioType.PCM);
 performAPT.setMuteAudio(false);
 
+sdlManager.addOnRPCNotificationListener(FunctionID.ON_AUDIO_PASS_THRU, new OnRPCNotificationListener() {
+    @Override
+    public void onNotified(RPCNotification notification) {
+        OnAudioPassThru onAudioPassThru = (OnAudioPassThru) notification;
+        // Process audio data
+    }
+});
+
 sdlManager.sendRPC(performAPT);
```

Comment:
This also needs to handle the response like the iOS code

Reply:
Should be fixed now.